endpoints/openrtb2/auction_benchmark_test.go

```diff
@@ -155,7 +156,7 @@ func BenchmarkValidWholeExemplary(b *testing.B) {
 				AccountRequired:    test.Config.AccountRequired,
 			}
 
-			auctionEndpointHandler, mockBidServers, mockCurrencyRatesServer, err := buildTestEndpoint(test, cfg)
+			auctionEndpointHandler, mockBidServers, mockCurrencyRatesServer, err := buildTestEndpoint(&testing.T{}, test, cfg)
```

Comment:
The creation of a new testing object looks suspicious. Can you get this from the benchmarking object?

Reply:
Done